front/src/common/BootstrapSNCF/NavBarSNCF.tsx

```diff
@@ -91,9 +92,10 @@ export default function LegacyNavBarSNCF({ appName, logo = logoOSRD }: Props) {
               </button>,
               <button
                 type="button"
-                className="btn-link text-reset"
+                className="user-settings-btn btn-link text-reset"
                 onClick={() => openModal(<UserSettings />)}
                 key="release"
+                disabled={pathname === '/'}
```

Comment:
Do you still want to keep that or Théo fixed the bug ? If we keep this change, can you add a comment "TODO  TS v2: remove this disabled" ? Thanks :)